### DISCUSSION TITLE
project need symulation

### DISCUSSION THREAD

**Discussion by fab1an2** [markqvist/Reticulum#218] [Ideas]
It seems to me that the project requires a simulator. I imagine it as a 2D map. Each node is in a specific location (which may change over time) and may or may not communicate with other nodes. Just like in the real world. Here is an obstacle, here is a block, a wall. 

Is there any way to run several thousand reticullum clients in one machine?

**Comment by srem1** [comment]
similar this?
https://github.com/GUVWAF/Meshtasticator or better

**Reply by sp00ck** [reply]
interesting
